### PR TITLE
Updated ns-3 to version 3.27.

### PR DESCRIPTION
**Pull request by tnull**

### CODE REVIEW

File: docsets/ns-3/docset.json [hunk 1]
```diff
@@ -12,6 +12,10 @@
 		{ 
 			"version": "3.25",
 			"archive": "versions/3.25/ns-3.tgz"
+		},	
+		{ 
+			"version": "3.26",
+			"archive": "versions/3.26/ns-3.tgz"
```
Comment (Kapeli) [inline docsets/ns-3/docset.json]: Newer versions should be at the top of this list.

Also, you need to add the actual `tgz` for this version. Moving the `txt` file doesn't work.

**Comment by tnull** [comment]
Should be better now?

**Comment by Kapeli** [comment]
Thank you! 👍

**Comment by tnull** [comment]
I found a little bug in my docset (it used the old Info.plist and therefore referenced the old fallback url). I updated my branch to fix this.

**Pull request merged**